test/unit/gateways/forte_test.rb

```diff
@@ -61,6 +61,16 @@ def test_failed_purchase_with_echeck
     assert_equal 'INVALID CREDIT CARD NUMBER', response.message
   end
 
+  def test_successful_purchase_with_service_fee
+    response = stub_comms(@gateway, :raw_ssl_request) do
+      @gateway.purchase(@amount, @credit_card, @options)
+    end.respond_with(MockedResponse.new(successful_purchase_response))
```

Comment:
How would we feel about adding a `successful_purchase_response_with_service_fee`, which we could crib from https://restdocs.forte.net/?version=latest#c8c05c2d-b18e-466a-bcf0-178a4e165895?

Reply:
And done!